### PR TITLE
Modify provision.sh, Makefile and update README

### PR DESCRIPTION
**Pull request by danialmalik**
Update README:
    Update README according to Edly requirements.

Makefile:
    Change "requirements" rule to install php and composer

provision.sh:
    remove provision-wordpress.sh

provision-wordpress.sh:
    add "#!/bin/bash" at start of file.

### CODE REVIEW

**Review by etechdevai — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -79,36 +69,59 @@ you should configure Docker with a sufficient
...
+
+    mkdir edly
+    cd edly
+
+2. Make a ``virutalenv`` for edly and activate it.
+
+.. code:: sh
 
-       make requirements
+    virtualenv -p python venv
```
Comment (zubair-arbi) [inline README.rst]: use python3

File: README.rst [hunk 2]
```diff
@@ -119,36 +132,156 @@ a minimum of 2 CPUs and 6GB of memory works well.
...
+    composer install
 
+    cd /var/www/html/wp-content/themes/edly-wp-theme
+    composer install
+
+3. Change permissions inside docker container.
+
+.. code:: sh
+
+    chmod 0777 /var/www/html/wp-content -R
```
Comment (zubair-arbi) [inline README.rst]: I believe we don't need this much open permissions. Try using following command:
```chown -R www-data:www-data wp-content```

File: README.rst [hunk 3]
```diff
@@ -119,36 +132,156 @@ a minimum of 2 CPUs and 6GB of memory works well.
...
+    ./provision-wordpress.sh
+
+
+
+
+Setup Edly Open edX theme
+-------------------------
+
+1. Go do the edly directory.
+2. Run ``sudo chmod -R 777 src/``
```
Comment (zubair-arbi) [inline README.rst]: I don't think we need to set these permissions. Try setting up edly devstack without this step.

File: README.rst [hunk 4]
```diff
@@ -119,36 +132,156 @@ a minimum of 2 CPUs and 6GB of memory works well.
...
+
+4. Edit the ``/edx/app/edxapp/lms.env.json`` file in the docker container to the following values.
+
+.. code:: json
+
+    "ENABLE_COMPREHENSIVE_THEMING": true,
+
+    "COMPREHENSIVE_THEME_DIRS": [
+        "/edx/app/edxapp/edx-platform/themes",
+        "/edx/src/edly-edx-themes"
```
Comment (zubair-arbi) [inline README.rst]: Remove this. With the change I made https://github.com/edly-io/devstack/pull/7, `st-lutherx` is automatically shared in the `/edx/app/edxapp/edx-platform/themes`. So remove this line.

File: README.rst [hunk 5]
```diff
@@ -119,36 +132,156 @@ a minimum of 2 CPUs and 6GB of memory works well.
...
+        Password: edx
+        Email: edx@example.com
+
+6. Click Install and the login with the same credentials.
+
+7. Run wordpress provsion.
+
+.. code:: sh
+
+    ./provision-wordpress.sh
```
Comment (zubair-arbi) [inline README.rst:220]: Also add steps to import `wordpress dump`, acf fields etc.

**Review by zubair-arbi — CHANGES_REQUESTED** [review]

File: README.rst [hunk 6]
```diff
@@ -119,36 +133,207 @@ a minimum of 2 CPUs and 6GB of memory works well.
...
+    sudo apt-get install php7.2
+    curl -s https://getcomposer.org/installer | php
+    sudo mv composer.phar /usr/local/bin/composer
+
+    For wordpress container
+
+.. code:: sh
+
+    make wordpress-shell
+    apt-get install php7.2
```
Comment (imhassantariq) [inline README.rst]: This container already contains php7.3 in it so we don't need it to install.

File: README.rst [hunk 7]
```diff
@@ -119,36 +133,207 @@ a minimum of 2 CPUs and 6GB of memory works well.
...
+        Password: edx
+        Email: edx@example.com
+
+6. Click Install and the login with the same credentials.
+
+7. Run wordpress provsion.
+
+.. code:: sh
+
+    ./provision-wordpress.sh
```
Comment (imhassantariq) [inline README.rst:220]: While provision we face several issues:
 - Permission denied Exception for theme and plugin folder this command outside helped:
`sudo chmod a+rw ../edly-wp-plugin -R`
`sudo chmod a+rw ../edly-wp-theme -R`

- Another issue is php7.2-xml is not install on my host machine so we installed it on our own using this command:
`sudo apt-get install php7.2-xml`

@tehreem-sadat Faced the same issues.

**Review by zubair-arbi — APPROVED** [review]
Thanks for updating the setup guide 👍

**Pull request merged**